Gems/EMotionFX/Code/Tests/UI/SkeletonOutlinerTestFixture.cpp

```diff
@@ -0,0 +1,175 @@
+
+        // while (true)
+        QApplication::processEvents();
+    }
+
+    /*
+        Test Cases
+     */
+    TEST_F(SkeletonOutlinerTestFixture, AddClothCollider)
+    {
+        SetUpPhysics();
+
+        // Find the 3rd joint after the RootJoint in the TreeView and select it
+        SelectIndexes(m_indexList, m_treeView, 3, 3);
+
+        AddColliderViaAddComponentButton("Add Cloth Collider", "Sphere");
+
+        ShowJointPropertyWidget();
+
+        EXPECT_TRUE(ColliderHelpers::NodeHasClothCollider(m_indexList[3]));
+    }
+
+    TEST_F(SkeletonOutlinerTestFixture, ChangeClothColliderValue)
+    {
+        SetUpPhysics();
+
+        AddColliderViaAddComponentButton("Add Cloth Collider", "Capsule");
+
+        // Check the node is in the ragdoll
+        EXPECT_TRUE(ColliderHelpers::NodeHasClothCollider(m_indexList[3]));
+
+        // get the widget
+        //auto* widget = GetJointPropertyWidget()->findChild<ClothJointWidget*>("EMotionFX.ClothJointWidget");
+        auto* widget = GetJointPropertyWidget();
+
+        // get a value widget
+        auto propertyEditor = widget->findChild<AzToolsFramework::ReflectedPropertyEditor*>("PropertyEditor");
+
+        // Get list of all PropertyRowWidgets (and their InstanceDataNodes)
+        const auto list = propertyEditor->GetWidgets();
+        ASSERT_GT(list.size(), 0) << "Did not find any PropertyRowWidgets";
+
+        // Look for PropertyRowWidget for "Name"
+        AzToolsFramework::PropertyRowWidget* propertyRow = nullptr;
+        for (const auto& item : list)
+        {
+            if (item.second->objectName() == "Height")
+            {
+                propertyRow = item.second;
+            }
+        }
+
+        // change it
+        auto *lineEdit = static_cast<AzToolsFramework::PropertyDoubleSpinCtrl*>(propertyRow->GetChildWidget());
+        //AZ_Assert(lineEdit) << "Did not find Editing handle";
+        lineEdit->setValue(3.89);
+        lineEdit->editingFinished();
+
+        // watch the world burn
+        ShowJointPropertyWidget();
+        // We did not crash, at least
+    }
+/*
+   TEST_F(SkeletonOutlinerTestFixture, CopyAndPaste)
+   {
+       SetUpPhysics();
+
+        // create a cloth collider to copy it
+        AddColliderViaAddComponentButton("Add Cloth Collider", "Capsule");
+
+        // copy it
+        //   ColliderWidget::OnCopyCollider();
+        //  find "Copy collider" action in cloth collider
+        auto* jointWidget = GetJointPropertyWidget()->findChild<ClothJointWidget*>();
+        auto* colliderContainerWidget = jointWidget->findChild<ColliderContainerWidget*>();
+        emit colliderContainerWidget->CopyCollider(0);
+
+        //auto* action = GetNamedAction(jointWidget, "Copy collider");
+        //action->trigger();
+        //   trigger action
+
+        // AddColliderViaAddComponentButton("Paste as Hit Detection Collider")
+        ShowJointPropertyWidget();
+       EXPECT_EQ(0, 1) << "This test is not implemented";
+   }
+ */
```

Comment:
We should delete this, or if the test is currently failing we could submit it as disabled with a ticket for fixing it up later.

Reply:
https://github.com/o3de/o3de/compare/065968e6435076ac5c075addcfed88df5b2a8903..79144bc7f76fab3f32d20bdd97bb74727104879e